### DISCUSSION TITLE
Image on the card gets cropped on the causes page

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#146] [Feedback]
![image](https://user-images.githubusercontent.com/101787508/162663975-a31ec4fb-5cfa-4c37-a8b3-ef1728cf2e45.png)

on the causes page the image is cropped from the sides

![image](https://user-images.githubusercontent.com/101787508/162664251-7545954c-6142-4d5a-93df-6b3e9d86c312.png)

**Comment by an-nadaa** [comment]
This is Expected behaviour. as mentioned previously, currently the algorithm is that it covers the area and takes the center.  So we need to choose good images that have the subjects centred. like the masjid picture or the orphans one.
